packages/react/src/index.ts

```diff
@@ -4,6 +4,7 @@
  */
 
 /* Append here */
+export * from './common/layout'
```

Comment:
Do we need to export utils here? We do not do this for the other one (keyboard focus) and it works fine.